### PR TITLE
feat(sizing): Syncronize sizing between Figma and React components

### LINKED ISSUE #1075: Syncronize sizing between Figma and React components
Update our components to use the same sizing values as in Figma. 

Let our our icons be relative to font size using `em` values. 

Some of our "input" components `Textfield`, `Textarea` etc are using `em` for, see if we can use our sizing tokens here.

### PR DESCRIPTION
**Pull request by Barsnes**
Resolves #1075

### CODE REVIEW

**Comment by mimarz** [comment]
Make sure to test for this use-case #1042

**Comment by Barsnes** [comment]
Waiting on @mrosvik's changes to `HelpText` for this to be moved to review. #1107

**Comment by mrosvik** [comment]
@Barsnes 

`Small`, `medium` and `large` sizes is now added to both `HelpText` and `Popover` in Figma ✅

https://www.figma.com/file/vpM9dqqQPHqU6ogfKp5tlr/Felles-komponenter?type=design&node-id=20759%3A80130&mode=design&t=pFMkxezHjX6MVQWE-1

**Comment by mrosvik** [comment]
@Barsnes 

`Small`, `medium` and `large` sizes is now added to both `HelpText` and `Popover` in Figma ✅

https://www.figma.com/file/vpM9dqqQPHqU6ogfKp5tlr/Felles-komponenter?type=design&node-id=20759%3A80130&mode=design&t=pFMkxezHjX6MVQWE-1

**Review by mimarz — CHANGES_REQUESTED** [review]
Not sure if this was intentional or not but looks like you missed to update the `padlock` unit size from `rem` to `em` on several other "form" components, `Textfield`, `Textarea` and `NativeSelect`

You have not done anything with the heights of `NativeSelect` or `Textfield` so the bug #1042 is not fixed which I was under the impression we wanted to fix by doing this PR also

File: packages/react/src/components/Pagination/Pagination.module.css [hunk 1]
```diff
@@ -41,8 +45,9 @@
   --fdsc-pagination-ellipsis-width: var(--fds-sizing-10);
 }
 
-.listitem.compact {
-  --fsdc-pagination-listitem-margin: var(--fds-spacing-1);
+.listitem.large {
```
[the remark this answers is not in the document]
Comment (Barsnes) [reply packages/react/src/components/Pagination/Pagination.module.css:48]: Were we not supposed to remove `compact`?

[the remark this answers is not in the document]
Comment (Thuneer) [reply packages/react/src/components/Pagination/Pagination.module.css:48]: Yes, the compact prop is needed for the pagination component to fit horizontally on the smallest mobile devices. From testing it has to be smaller than 352px in width.

[the remark this answers is not in the document]
Comment (mimarz) [reply packages/react/src/components/Pagination/Pagination.module.css:48]: Added the compact class back.

**Review by mimarz — APPROVED** [review]
LGTM 👍 

We still have two bugs related to sizing, #1042 and #1146 which we will solve in separate PRs

**Pull request merged**